### PR TITLE
Connect PublicProfilePage to redux and update data models

### PR DESCRIPTION
**Pull request by luoto**
More updates needed to model, will continue on another branch.

### CODE REVIEW

File: src/ProjectPortalPage/Chat.tsx [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-class Chat extends React.PureComponent<{}, {}> {
+class Chat extends React.PureComponent<
+  { comments: Array<string> | undefined | string },
+  {}
+> {
   render() {
     return (
       <React.Fragment>
-        <Messages />
+        <Messages {...this.props} />
```
Comment (serpient) [inline src/ProjectPortalPage/Chat.tsx]: what does this.props refer to?

File: src/actions/projectActions.ts [hunk 2]
```diff
@@ -20,6 +21,19 @@ export function getProjects(): (dispatch: Dispatch<Action>) => void {
   };
 }
 
+export function getProject(
```
Comment (serpient) [inline src/actions/projectActions.ts]: lol i have a getProject action as well. will have to do a merge conflict once my stuff is in

File: src/utils/apiService.ts [hunk 3]
```diff
@@ -325,6 +325,41 @@ function getProjects(): Promise<Array<Project>> {
   });
 }
 
+function getProject(projectId: string) {
```
Comment (serpient) [inline src/utils/apiService.ts]: was there a reason why the response is pre-filled?

File: src/ProjectPortalPage/Chat.tsx (hunk 1 above)
Comment (luoto) [reply src/ProjectPortalPage/Chat.tsx]: It just passing through what `Chat` receives.

File: src/actions/projectActions.ts (hunk 2 above)
Comment (luoto) [reply src/actions/projectActions.ts]: That should be okay, I'm sure they will be doing the same thing!

File: src/utils/apiService.ts (hunk 3 above)
Comment (luoto) [reply src/utils/apiService.ts]: Just for quick testing, will implement actual api call in the next steps.

**Pull request merged**